examples/src/main/java/io/dapr/examples/state/StateClient.java

```diff
@@ -70,10 +70,10 @@ public static void main(String[] args) throws Exception {
       operationList.add(new TransactionalStateOperation<>(TransactionalStateOperation.OperationType.UPSERT,
               new State<>(secondState, SECOND_KEY_NAME, "")));
 
-      client.executeTransaction(STATE_STORE_NAME, operationList).block();
+      client.executeStateTransaction(STATE_STORE_NAME, operationList).block();
```

Comment:
Please, update README.md for all affected examples. We copy and paste code from here there.

Reply:
Done